client/python/tensorflow-serving-api/samples/requirements.txt

```diff
@@ -1,2 +1,3 @@
 tensorflow-serving-api<=2.12.2,>=2.10.1
 protobuf<=3.20.3
+tensorflow==2.13.1
```

Comment:
Why this particular version? 
Don't we want to also change upper bound for `tensorflow-serving-api` so in default scenario they match?

Reply:
This is the latest version that does not cause any conflicts.
Bumped tensorflow-serving-api to 2.13.1